### ISSUE TITLE
Support key combination triggers

### ISSUE THREAD

**Issue by fedeantuna** [fedeantuna/warm-corners#2]
**Description**

System must be capable of processing `Key Combination` triggers.

When the mouse cursor over a screen corner, if configured correctly, the system must execute a specified combination of keystrokes.
 
**Proposed Solution**

When the user moves the mouse cursor over a screen corner, the key combination configured for that particular corner gets executed. In order to decide if it gets executed, the system will evaluate if the mouse cursor is in the actual corner that the user specified.

The evaluation to decide whether or not the mouse cursor is in the corner that the user specified is by taking into account the display resolution.

- Top left corner: (0, 0)
- Top right corner: (DISPLAY_WIDTH, 0)
- Bottom right corner: (DISPLAY_WIDTH, DISPLAY_HEIGHT)
- Bottom left corner: (0, DISPLAY_HEIGHT)

The triggers should know the screen corner where they have to run and the key combination that the user wants to execute.

In summary, to determine whether or not to execute a key combination, the system will need to know the screen corner that the user wants the trigger to run on and the current position of the mouse cursor.

**Test Cases**

- Given a list of key combination triggers, run only the ones that are configured for X corner when the mouse is in X corner
